packages/ncurses.rb

```diff
@@ -3,38 +3,68 @@
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/ncurses-6.1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/ncurses-6.1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/ncurses-6.1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ncurses-6.1-chromeos-x86_64.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/ncurses-6.1-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/ncurses-6.1-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/ncurses-6.1-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ncurses-6.1-0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '4e33d4fc68ea5b47bae07aa710b1f801fd80ba85a02c0060c6f157100acadaf4',
-     armv7l: '4e33d4fc68ea5b47bae07aa710b1f801fd80ba85a02c0060c6f157100acadaf4',
-       i686: '16a5f5c03a4e9591146dc56a1ed4b017c09ad9d72da38d104ce282a3d25ef9fd',
-     x86_64: '1eab39eee49e16799ad61def29e69c50a7ea39b04a8da2ff4f4ada033e81273c',
+    aarch64: 'a12e70f779b2375912a5373b5f320c7b4bef8ddcba5144709e2aed67f8011afe',
+     armv7l: 'a12e70f779b2375912a5373b5f320c7b4bef8ddcba5144709e2aed67f8011afe',
+       i686: 'a89d5d79d91c8717cfecf7ebbebae6479f8cded2884798643d061605e449c1df',
+     x86_64: 'fcaf728604029920a14f27f64f498dfc8a3586bc0426ca6a890adf925e39e204',
   })
 
   depends_on 'diffutils' => :build
-  depends_on 'ncursesw'
 
   def self.build
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           '--without-normal',
-           '--with-shared',
-           '--with-cxx-shared',
-           '--without-debug'
-    system 'make'
+    # build libncursesw
+    system "mkdir -p ncursesw_build"
+    Dir.chdir("ncursesw_build") do
+      system "../configure",
+             "--prefix=#{CREW_PREFIX}",
+             "--libdir=#{CREW_LIB_PREFIX}",
+             "--without-normal",
+             "--with-shared",
+             "--with-cxx-shared",
+             "--without-debug",
+             "--enable-pc-files",
+             "--with-pkg-config-libdir=#{CREW_LIB_PREFIX}/pkgconfig",
+             "--enable-widec"
+      system "make"
+    end
+
+    # build libncurses
+    system "mkdir -p ncurses_build"
+    Dir.chdir("ncurses_build") do
+      system "../configure",
+             "--prefix=#{CREW_PREFIX}",
+             "--libdir=#{CREW_LIB_PREFIX}",
+             "--without-normal",
+             "--with-shared",
+             "--with-cxx-shared",
+             "--without-debug",
+             "--enable-pc-files",
+             "--with-pkg-config-libdir=#{CREW_LIB_PREFIX}/pkgconfig",
+             "--disable-db-install",
+             "--without-manpages",
+             "--without-progs",
+             "--without-tack"
+      system "make"
+    end
```

Comment:
Oh, I see what you are doing now.  You want to combine ncursesw in with the ncurses package?

Reply:
Yes.

Reply:
Since they come from the same source code

Reply:
Yes, that makes a lot of sense.  I like the way you think. :)